### DISCUSSION TITLE
Need a way to integrate AWS Event bridge with Third party SaaS Application (In My Case New Relics).

### DISCUSSION THREAD

**Discussion by chaurasiavishal** [serverless/serverless#12039] [Q&A]
### Is there an existing issue for this?

- [X] I have searched existing issues, it hasn't been reported yet

### Use case description

I have an Lambda which is invoked using by an event generated in new relics and, The New relic send the event to AWS Event Bridge and this Event bridge is used to invoke the lambda, the issue I am facing is when I am deploying the serverless YAML file, in which source is mentioned as event bridge ( aws.partner/newrelic.com ), This source throughs error for authentication.

The Deployment is carried out using Github Actions.

I checked for a way to authenticate this, but I was unable to find one, so I am requesting a feature where we can authenticate in between deployment for that specific function, only.

### Proposed solution (optional)

I am requesting a feature where we can authenticate in between deployment for that specific function, only.

**Comment by medikoo** [comment]
@vishaliamops this issue tracker is for bug reports only. I think best is to ask such questions on Stack Overflow.

I'm moving this to discusionss

**Reply by chaurasiavishal** [reply]
Sure @medikoo I will looking, forward to discussion.

Thank You for responding.
